site/assignments/zombies/zombies.rst

```diff
@@ -1,33 +1,197 @@
 *****************
-Oh crap, Zombies!
+Zombie Infections
 *****************
 
 * **Worth**: 10%
-* **DUE**: Oct 25, at 11:55pm; submitted on MOODLE.
-* Starting point: :download:`asn2.ipynb <asn2.ipynb>` OR :download:`asn2.py <asn2.py>`
+* **Due**: TBD 2022, at 11:55pm; submitted on MOODLE
+* **Files**: :download:`asn2.ipynb <asn2.ipynb>`/:download:`asn2.py <asn2.py>`
+
 
 .. warning::
 
-   For this assignment, you may **not** work with anyone else. You need to work independently. By all means communicate with each other at a high level (that's good), but your code needs to be your own here. 
+    For this assignment, you will need to use special Python packages (networkx, matplotlib). The good news is that if
+    you are using Colab you do not need to do anything special. If, however, you are using a local Python interpreter,
+    you may need to install the packages yourself.
+
+
+Task
+====
+
+
+Provided Files
+==============
+
+
+Part 0 --- Read the Assignment
+==============================
+
+Read the assignment description in its entirety before starting.
+
+
+Part 1 --- Uploading Files to Colab
+===================================
+
+
+Part 2 --- Read Over Provided Code
+==================================
+
+
+Part 3 --- Check if a City is Infected
+======================================
+
+
+Part 4 --- Get a City's Neighbours
+==================================
+
+
+Part 5 --- Infecting and Curing a City
+======================================
+
+
+Part 6 --- How Many Cities are Infected
+=======================================
+
+* Both ``number_of_cities_infected`` and ``is_world_completely_infected`` here
+
+
+Part 7 --- Simulation Step
+==========================
+
+
+Part 8 --- Simulate Infections Disease Scenario
+===============================================
+
+
+Part 9 --- Using Your Code and Plotting
+=======================================
+
+
+Part 10 --- Testing
+===================
+
+
+Some Hints
+==========
+
+* Work on one method at a time
+* Get each method working perfectly before you go on to the next one
+* Test each method as you write it
```

Comment:
At this stage, they are writing functions, not methods.

Reply:
:+1: